CHANGELOG.md

```diff
@@ -6,20 +6,24 @@
 
 ### Major features and improvements
 
+- StellarGraph is now available as [a conda package on Anaconda Cloud](https://anaconda.org/stellargraph/stellargraph) [\#516](https://github.com/stellargraph/stellargraph/pull/516)
 - New algorithms:
   - Cluster-GCN: an extension of GCN that can be trained using SGD, with demo [\#487](https://github.com/stellargraph/stellargraph/issues/487)
+  - `FullBatchLinkGenerator` has been added to do link prediction using full-batch algorithms like GCN, GAT, APPNP and PPNP [\#543](https://github.com/stellargraph/stellargraph/pull/543)
```

Comment:
I'm not sure if we should consider this a "new algorithm" or not.

Reply:
If you rephrase it to say something like "Link prediction for full batch models. FullBatchLinkGenerator..." then it will fit under the new algorithms banner.

Reply:
```suggestion
  - Link prediction for full-batch models: `FullBatchLinkGenerator` allows doing link prediction with algorithms like GCN, GAT, APPNP and PPNP [\#543](https://github.com/stellargraph/stellargraph/pull/543)
```